webapp-django/crashstats/crashstats/templates/crashstats/exploitability_report.html

```diff
@@ -0,0 +1,81 @@
+{% extends "crashstats_base.html" %}
+
+{% block page_title %}Exploitable Crashes{% endblock %}
+
+{% block site_js %}
+{{ super() }}
+{% compress js %}
+<script src="{{ static('crashstats/js/socorro/bugzilla.js') }}"></script>
+{% endcompress %}
+{% compress js %}
+<script src="{{ static('crashstats/js/socorro/exploitability_report.js') }}"></script>
+{% endcompress %}
+{% endblock %}
+
+{% block site_css %}
+{{ super() }}
+<style>
+table.data-table td.ajax-loading {
+    text-align: center;
+    padding: 100px 0;
+    background-color: white;
+}
+.pagination {
+    display: none; /* gets displayed after first ajax load */
+}
+.pagination a.impossible {
+    text-decoration: none;
+    color: #666;
+    cursor: default;
+}
+</style>
+{% endblock %}
+
+
+{% block content %}
+<div id="mainbody">
+    <div class="page-heading">
+        <p class="old-new-report-link">
+            <a href="{{ old_link_url }}">
+                View the <b>old</b> <i>Exploitable crashes</i> report
+            </a>
+        </p>
+        <h2>Exploitable Crashes for {{ product }}
+        {% if version %}{{ version }}{% endif %}</h2>
+    </div>
+
+    <div class="panel">
+        <div class="body">
+
+            <div class="pagination">
+                <a href="#" class="first">&lt;&lt;First</a>
+                <a href="#" class="previous">&lt;Previous</a>
+                <a href="#" class="next">Next&gt;</a>
+                <a href="#" class="last">Last&gt;&gt;</a>
+            </div>
+
+            <table class="builds data-table">
+                <thead>
+                    <tr>
+                        <th>Signature</th>
+                        <th>Bugzilla IDs</th>
+                        <th>Count HIGH</th>
+                        <th>Count MEDIUM</th>
+                        <th>Count LOW</th>
+                        <th>Count NONE</th>
+                    </tr>
+                </thead>
+                <tbody>
+                    <tr>
+                        <td colspan="6" class="ajax-loading">
+                            <!-- this gets replaced by jQuery loading from AJAX -->
+                            <img
+                                src="{{ static('img/icons/ajax-loader.gif') }}">
```

Comment:
Why JS and not jinja? Jinja is what we use everywhere else, and recently we even removed mustache in favor of jinja, so I don't get why going back to doing templating in JavaScript.